docs/src/changelog.md

```diff
@@ -17,7 +17,8 @@ If your currently installed *Quality-time* version is not v5.13.0, please first
 ### Fixed
 
 - In the dropdown of the "Add metric" button, keep the two checkboxes next to each other regardless of the width of the menu. Fixes [#8745](https://github.com/ICTU/quality-time/issues/8745).
-- The icon of the trend graph tab would not be shown. Closes [#8822](https://github.com/ICTU/quality-time/issues/8822).
+- The icon of the trend graph tab would not be shown. Fixes [#8822](https://github.com/ICTU/quality-time/issues/8822).
+- Prevent zombie browser processes in the renderer component by starting Chromium only once and opening and closing a page for each render. Fixes [#8979](https://github.com/ICTU/quality-time/issues/8979).
```

Comment:
"and opening and closing" sounds superfluous, should be sufficient to say "and closing"